src/main/java/com/gmail/goosius/siegewar/SiegeController.java

```diff
@@ -293,5 +294,15 @@ public static void setSiege(Town town, boolean bool) {
 		}
 		return result;
 	}
+
+	public static List<Siege> getSiegesAt(Location location) {

```

Comment:
:construction: 
- As I understand it, you are looking for 'active' sieges
- Thus please rename this method to "getActiveSiegesAt"
- Also add logic to only add active sieges